PROTOCOL.md

```diff
@@ -265,19 +265,25 @@ The following is an example `metaData` action:
 <!-- TODO: forward references configuration options -->
 
 ### Add File and Remove File
-The `add` and `remove` actions are used to modify the data in a table by adding or removing individual data files respectively.
+The `add` and `remove` actions are used to modify the data in a table by adding or removing individual _logical files_ respectively.
```

Comment:
```suggestion
The `add` and `remove` actions are used to modify the data in a table by adding or removing individual _logical data files_ respectively.
```

Reply:
No, entire logical files, not data files alone. "Logical file" is "data file + DV (if any)" and that is the unit being modified here.